src/views/Lending/CoolerV2/components/CreateOrRepayLoanV2.tsx

```diff
@@ -242,6 +271,14 @@ export const CreateOrRepayLoanV2 = ({ setModalOpen, modalOpen, loan, isRepayMode
                 );
               }
 
+              if (exceedsUndelegatedAmount) {
+                return (
+                  <PrimaryButton fullWidth disabled>
+                    You must undelegate before withdrawing
```

Comment:
I think you can make this a bit easier: link to the manage delegations page, and give them a target amount to reduce delegations to